src/libs/highlevel/README.md

```diff
@@ -30,15 +30,13 @@ if (elektra == NULL)
 }
 
 int myint = elektraGetLong (elektra, "myint");
+printf ("got long %d\n", myint);
```

Comment:
is "Long" guaranteed to be an int?

Reply:
"Long" is guaranteed to be 32 bits that should fit into an int on most modern systems. But you are right using `kdb_long_t` would be safer and better for the example.